example/basic/server.js

```diff
@@ -13,7 +13,7 @@ const routes = [
   ['/', html],
   ['/404', html],
   ['/bundle.css', css],
-  ['bundle.js', js]
+  ['/bundle.js', js]
```

Comment:
I just noticed that this path ought to start with a slash, at least for consistency.